lib/src/component_declaration/flux_component.dart

```diff
@@ -50,14 +50,18 @@ mixin FluxUiPropsMixin<ActionsT, StoresT> on UiProps implements FluxUiProps<Acti
   /// The prop defined by [ActionsT] that holds all [Action]s that
   /// this component needs access to.
   ///
+  /// ** Required **
+  ///
   /// There is no strict rule on the [ActionsT] type. Depending on application
   /// structure, there may be [Action]s available directly on this object, or
   /// this object may represent a hierarchy of actions.
   @override
-  ActionsT? actions;
+  late ActionsT actions;
```

Comment:
Hmmm thinking it a little more, making `actions` required might be a little tricky.

While it'll be present in most cases, there are usages that only wire up a component to redraw on the store, because the actions aren't used. Internally, it looks like ~5% of flux component usages had a store with no actions.

So our options are either:
- Make it optional
- Make it required, and plan on migrating consumers to also set `..actions =` (either to an actions class that will be unused, or to set `..actions = null` and potentially update the `TActions` generic of the consuming component to be `with FluxUiPropsMixin<MyStore, Null>`)

5% isn't a lot, so I'm leaning toward making it required, but it'll be a little bit of a pain to update those usages.

Thoughts?

Reply:
I like making it required.